apps/api/src/app/content-templates/usecases/compile-template/compile-template.spec.ts

```diff
@@ -61,10 +62,29 @@ describe('Compile Template', function () {
       })
     );
 
+    it('should allow the user to specify handlebars helpers'),
+      async function () {
+        const result = await useCase.execute(
+          CompileTemplateCommand.create({
+            templateId: 'custom',
+            data: {
+              branding: {
+                color: '#e7e7e7e9',
+              },
+              message: 'hello world',
+            },
+            customTemplate: '<div>{{titlecase message}}</div>',
+          })
+        );
+      };
+
     expect(result).to.contain('Hello TESTTTT content');
     expect(result).to.not.contain('{{#each blocks}}');
     expect(result).to.not.contains('ff6f61');
     expect(result).to.contain('#e7e7e7e9');
     expect(result).to.contain('Button content of text');
+    expect(result).to.contain('{{titlecase message}}');
```

Comment:
These should be the actual result you are expecting. Like: <div>Hello World</div>